docs/developing.rst

```diff
@@ -373,6 +373,64 @@ workflow, with jobs defined within a job or job function being created at
 run time.
 
 
+Developing pipelines with toil-lib
+----------------------------------
+
+`toil-lib`_ provides many functions and job functions that are useful in
+developing pipelines in Toil. Installation instructions for toil-lib are
+available `here`_.
+
+.. _toil-lib: https://github.com/BD2KGenomics/toil-lib/
+
+.. _here: https://github.com/BD2KGenomics/toil-lib/blob/master/docs/installation.rst
+
+toil-lib packages and functions can be imported to a Toil pipeline like any
+Python package. Jobs from toil-lib can then be added to workflows staticly.
```

Comment:
Also, this text makes it look like jobs from `toil_lib` are limited to being strung up in a static DAG.